src/main/java/net/minecraftforge/common/brewing/BrewingRecipeRegistry.java

```diff
@@ -168,4 +167,80 @@ public static boolean isValidInput(ItemStack stack)
     {
         return Collections.unmodifiableList(recipes);
     }
+
+    /**
+     * Returns an unmodifiable list of all fuels in the registry.
+     */
+    public static List<FuelEntry> getFuels()
+    {
+        return Collections.unmodifiableList(fuels);
+    }
+
+    /**
+     * Adds a new fuel for brewing stands.
+     */
+    public static void addFuel(ItemStack item, int fuelValue)
+    {
+        fuels.add(new FuelEntry(item, fuelValue));
+    }
+
+    /**
+     * Check whether an item is a fuel of brewing stands.
+     */
+    public static boolean isFuel(@Nullable ItemStack item)
+    {
+        if (item == null)
+        {
+            return false;
+        }
+        for (FuelEntry e : fuels)
+        {
+            if (e.getItem().isItemEqual(item))
+            {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Gets the fuel value of an item.
+     */
+    public static int getFuelValue(@Nullable ItemStack fuel)
+    {
+        if (fuel == null)
+        {
+            return 0;
+        }
+        for (FuelEntry e : fuels)
+        {
+            if (e.getItem().isItemEqual(fuel))
+            {
+                return e.getFuelValue();
```

Comment:
he means it might be good to implement some sort of capability "can fuel"
and the stand checks for the cap's presence and how much it can provide